### ISSUE TITLE
Add GH action to protect from schema changes

### ISSUE THREAD

**Issue by prrao87** [superlinked/VectorHub#297]
## Issue

We need to add a GH action to protect against schema changes for the comparison table. Something that informs the user that any changes to the `vendor.schema.json` file by an external contributor would need to be issued as a separate PR (with no overlap with non-schema related PRs).

This is needed mainly because not all schema changes are necessary, and we'd need to discuss the impact of the schema change before affecting all vendors simultaneously.

**Comment by AruneshSingh** [comment]
On it

**Comment by msmygit** [comment]
@prrao87 will you be re-introducing the changes from https://github.com/superlinked/VectorHub/pull/295 which was merged earlier and subsequently got reverted?

**Comment by prrao87** [comment]
@msmygit that isn't my decision alone to make :). The team here need to discuss and we will keep you posted.

**Comment by ClaireSuperlinked** [comment]
@AruneshSingh is there an update on this?

**Comment by AruneshSingh** [comment]
Hey, we've started working on the action this week, I'll give a better estimate on when we can deliver it in a day. 

Regarding the columns that were added in the PR above (and subsequently removed), we've added it to our list of attributes for us to add. Due to our limited resources we are prioritising attributes/stats that are highly requested _and_ can be filled for most of the vendors. @msmygit It would be great to signal if such attributes are requested by more vendors/help understand the space better.
